setup.py

```diff
@@ -29,7 +29,7 @@ def read_long_description_from_readme():
 
 setup(
     name="openstf_dbc",
-    version="1.2.0",
+    version="1.3.0",
```

Comment:
Would it make sense to increase the major version number?
Since some previously existing functions have been removed, this could cause issues in e.g. icarus-ops-gui?

Reply:
Good point, I did made a couple small changes which are not backward compatible. Semver also says you should increment the major version in that case. On the other hand we could also just update the minor like I did and ensure backward compatibility. For example, by allowed both old and new names untill the next major release:

```python
    def get_apx(self, datetime_start, datetime_end, forecast_resolution=None):
        warnings.warn(
            "Please use the get_electricity_price method instead of get_apx",
            DeprecationWarning,
        )
        return self.get_electricity_price(
            datetime_start, datetime_end, forecast_resolution
        )
```

What would you prefer?

Reply:
Let's go major!

Reply:
done